test/common/testlib.py

```diff
@@ -2131,6 +2131,22 @@ def setup_provisioned_hosts(self, disable_preload: bool = False):
             m.execute(f"hostnamectl set-hostname {name}")
             if disable_preload:
                 self.disable_preload("packagekit", "playground", "systemd", machine=m)
+    def reboot(self,timeout_sec):
+        self.machine.allow_restart_journal_messages()
+        self.machine.reboot(timeout = timeout_sec)
+    
+    def reboot(self):
+        self.machine.allow_restart_journal_messages()
+        self.machine.reboot()
+    
+    def wait_reboot(self, timeout_sec): 
+        self.machine.allow_restart_journal_messages()
+        self.machine.wait_reboot(timeout_sec = timeout_sec)
+
+    def wait_reboot(self): 
```

Comment:
Please remove duplicate.

Reply:
This is great, but nothing actually uses it. Please adjust the existing tests to use the new self.wait_reboot. For example testTracer in test/verify/check-packagekit.

    Adjust the usage of m.wait_reboot in test/verify/check-packagekit
    Prepare a test virtual machine: ./test/image-prepare -q fedora-37
    Run the test: TEST_OS=fedora-37 ./test/verify/check-packagekit -vst

If this works as expected (test passes), adjust all other occurrences of m.wait_reboot in test/verify.

See original comment at https://github.com/cockpit-project/cockpit/pull/18431/files#r1122949054